### ISSUE TITLE
Investigate discrepancy between COMET and BLEU scores

### ISSUE THREAD

**Issue by eu9ene** [mozilla/firefox-translations-models#104]
For some translators, we can see the opposite results for BLEU and COMET when compared to Bergamot.
This is mostly relevant for open-source translators.

For example, for cs-en we can see:

BLEU     argos -33% nllb -27%
COMET argos +47% nllb +51%

There might be a bug somewerhe.

BLEU prod:

<img width="1023" alt="Screenshot 2023-11-07 at 9 21 22 AM" src="https://github.com/mozilla/firefox-translations-models/assets/2486505/684b9113-21f7-4b06-a219-43a54ba2a4d1">

COMET prod:

<img width="1033" alt="Screenshot 2023-11-07 at 9 21 11 AM" src="https://github.com/mozilla/firefox-translations-models/assets/2486505/15d9a3bb-435f-404e-a56d-f74ec0e15ff8">

**Comment by eu9ene** [comment]
We recently updated COMET to 2.1.1. from 1.1.3. It uses different models. We should recalculate COMET scores for all languages for Bergamot, Microsoft and Google. I suggest implementing #91 first to make sure we don't translate everything every time we change metrics.

**Comment by marco-c** [comment]
I opened #107 to test the hypotesis that the score change is due to the update of Comet.

**Comment by marco-c** [comment]
Yes, we can see that the recalculated scores are indeed much higher than before.
